ibis/mapd/tests/conftest.py

```diff
@@ -23,6 +24,23 @@ def con():
     )
 
 
+@pytest.fixture(scope='module')
+def session_con():
+    con = pymapd.connect(
```

Comment:
Why use `pymapd` explicitly here instead of just passing in `sessionid` to `ibis.mapd.connect`?

Reply:
The idea here is to *generate* a valid `sessionid` using the underlying pymapd library, and then use it to make an `ibis` connection.

Reply:
Ah, I see. Is this the intended user-facing way to use the session id parameter?

Reply:
The intended user-facing way (as far as I understand it) is to have a session id from some other context, which is more or less unknown to us. This session has already been authenticated, and can be used in lieu of a password. The invocation of `pymapd` here should be considered an implementation detail for the test, not the intended way to use the parameter.